### PR TITLE
Implement RTI problem

### PR DESCRIPTION
**Pull request by tbhaxor**

### CODE REVIEW

File: inputs/rt.in [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+x1min     = -0.16666667 # minimum value of X1
+x1max     = 0.16666667  # maximum value of X1
+ix1_bc    = periodic    # inner-X1 boundary flag
+ox1_bc    = periodic    # outer-X1 boundary flag
+
+nx2       = 200         # Number of zones in X2-direction
+x2min     = -0.5        # minimum value of X2
+x2max     = 0.5         # maximum value of X2
+ix2_bc    = reflecting     # inner-X2 boundary flag
+ox2_bc    = reflecting     # outer-X2 boundary flag
```
Comment (pgrete) [inline inputs/rt.in:27]: FYI reflecting boundary conditions are not working and need to be merged, see #58
Realistically, this should happen later this week or early next week.

File: src/pgen/pgen.hpp [hunk 2]
```diff
@@ -121,4 +121,9 @@ void UserWorkBeforeOutput(MeshBlock *pmb, ParameterInput *pin);
 void Cleanup();
 } // namespace turbulence
 
+namespace rt {
+  using namespace parthenon::driver::prelude;
+  void ProblemGenerator(MeshBlock *pm, parthenon::ParameterInput *pin);
```
Comment (pgrete) [inline src/pgen/pgen.hpp:126]:
```suggestion
  void ProblemGenerator(MeshBlock *pmb, parthenon::ParameterInput *pin);
```

For consistency. (`*pm` usually refers to the mesh pointer)

File: src/pgen/rt.cpp [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                u(IM2,k,j,i) = 0.0;
+                u(IM3,k,j,i) *= (den*amp);
+                u(IEN,k,j,i) = (p0 + grav_acc*den*x3v)/gm1 + 0.5*SQR(u(IM3,k,j,i))/den;
+              }
+              break;
+            }
+        }
+      }
+    }
+  }
```
Comment (pgrete) [inline src/pgen/rt.cpp:94]: I don't see the deep copy that I mentioned in #69

Comment (tbhaxor) [reply src/pgen/rt.cpp]: Done

**Comment by tbhaxor** [comment]
@pgrete Once you have some output files

```
./build/bin/athenaPK -i ./inputs/rt.in  -d rt
python3 external/parthenon/scripts/python/packages/parthenon_tools/parthenon_tools/movie2d.py prim rt/*.phdf --output-directory movie
```

File: inputs/rt.in (hunk 1 above)
Comment (tbhaxor) [reply inputs/rt.in]: Changes to outflow for now

**Comment by tbhaxor** [comment]
Hi @pgrete Did you get time to check this. I have implemented Jet problem from athena codebase. Same behaviour. No data is shown in movie.

I reconfirmed using in KH problem in AthenaPK, the movie2d.py script works.

**Comment by pgrete** [comment]
Oh, wait, so you're using the `movie2d` script?
Could you please double check if you see sth in the outputs using a different viz tool (Paraview, Visit, or yt) so that we can narrow down the issue to the sim itself or the analysis.

**Comment by tbhaxor** [comment]
> Oh, wait, so you're using the `movie2d` script? Could you please double check if you see sth in the outputs using a different viz tool (Paraview, Visit, or yt) so that we can narrow down the issue to the sim itself or the analysis.

Yes I checked on render view, it is empty

**Comment by tbhaxor** [comment]
@pgrete ping 0x00;

**Comment by tbhaxor** [comment]
Closing this PR for now, as its idle. I will share if I found the solution

**Pull request closed without merging**